_spinnaker/armory_halyard.md

```diff
@@ -290,6 +290,65 @@ hal armory dinghy slack disable [parameters]
  * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
  * `--no-validate`: (*Default*: `false`) Skip validation.
 
+---
+## hal armory dinghy webhooksecrets <provider> [enable | disable]
+
+Enable/Disable Webhook secrets validations, only `github` provider is supported at this moment. If this option is enabled all github webhooks will be validated.   (Halyard >= ? add version)
+
+#### Usage
+```
+hal armory dinghy webhooksecrets <provider> [ enable | disable]
```

Comment:
Can you update the dinghy operator config in this page to include webhook secrets (https://docs.armory.io/spinnaker/armory_halyard/#equivalent-spinnakerserviceyml-config-for-spinnaker-operator-1)? we try to always include documentation for both halyard commands and its corresponding operator config

Reply:
Can you add a  `webhooksecrets` section to  the `dinghy` entry in https://github.com/armory/documentation/blob/master/_operator_reference/armory.md  also?

Reply:
@german-muzquiz Done!
@aimeeu Done!